docs/pages/usage/namespace_offloading.md

```diff
@@ -0,0 +1,162 @@
+```yaml
+clusterSelector:
+  nodeSelectorTerms:
+  - matchExpressions:
+    - key: liqo.io/type
+      operator: In
+      values:
+      - virtual-node
+``` 
+
+{{% notice info %}}
+In **Liqo 0.3**, the *namespaceOffloading* must contain the configuration at creation time. 
+If you want to modify its structure at runtime, you should delete the resource and recreate it with a new configuration. 
+This triggers the deletion of all remote "twin" namespaces and the creation of the new ones.
+{{% /notice %}}
+
+The *matchExpressions*, specified in the **clusterSelector** term, select labels attached to Liqo virtual nodes.
+
+More precisely, at Liqo installation time, you may identify a set of labels to expose the most relevant features of your cluster. 
+After the peering phase, the virtual node will expose those labels, enabling the possibility to select it during the offloading configuration. 
+Moreover, the scheduler will also be able to use these virtual nodes' labels to impose affinity/anti-affinity policies at run-time.
+
+It is worth noting that there is no restriction on the labels to choose. 
+Labels can characterize your clusters showing their geographical location, the underlying provider, or the presence of specific hardware devices.
+
+{{% notice tip %}}
+If you just want to create deployment topologies that include all available clusters you are not required to choose labels at installation time. 
+All virtual nodes expose the label `liqo.io/type = virtual-node` by default.
+{{% /notice %}}
+
+### NamespaceOffloading in quick offloading
+
+Also the *quick Offloading* relies on the namespaceOflloading resource. 
+In fact, when the label `liqo.io/enabled = true` is added to a namespace, this event triggers the creation of a default *namespaceOffloading* resource inside the namespace.
+
+The generated resource is exactly equal to the template [seen above](#custom-offloading), setting all fields to the default values:
+
+* **namespaceMappingStrategy**: *DefaultName*.
+* **podOffloadingStrategy**: *LocalAndRemote*.
+* **clusterSelector**: *all remote clusters selection*.
+
+{{% notice info %}}
+Using the *quick offloading*, it is not possible to customize the generated *namespaceOffloading* resource. 
+If a *namespaceOffloading* is already present in the namespace, you should remove the label first and then create the new resource.
+Removing the label is equivalent to removing the resource directly.
+{{% /notice %}}
+
+{{% notice tip %}}
+If you create a custom *namespaceOffloading* inside the namespace, the `liqo.io/enabled = true` label is not necessary.
+As previously said, this label is just a way to configure a *quick offloading*. 
+{{% /notice %}}
+
+### Check the namespaceOffloading status
+
+The Liqo controllers update the *namespaceOffloading* every time there is a change in the deployment topology. 
+The resource status provides different information:
+
+```bash
+kubectl get namespaceoffloading offloading -n test-namespace -o wide
```

Comment:
```suggestion
kubectl get namespaceoffloading offloading -n target-namespace -o wide
```

Is the name `offloading` mandatory? If so, it is not specified in the documentation.

Reply:
Yes it is mandatory, I have added a "note". Thank you